pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -224,6 +264,35 @@ func createDynamicMetadataMatcher(k, v string) *metadata.MetadataMatcher {
 			},
 		}
 	}
+	return stringMatcher
+}
+
+// createDynamicMetadataMatcher creates a MetadataMatcher for the given key, value pair.
+func createDynamicMetadataMatcher(k, v string) *metadata.MetadataMatcher {
+	keys, err := generateMetaKeys(k)
+	if err != nil {
+		return nil
+	}
+	forceRegexPattern := false
+	if k == attrSrcNamespace {
+		// Change the value to a regular expression to match the namespace part.
+		v = fmt.Sprintf(`*/ns/%s/*`, v)
+		forceRegexPattern = true
+	}
+
+	//Handle the claims under authDerivedClaims
```

Comment:
Add a white space between `//` and `Handle`

Reply:
Done.